backend/app/models/user.rb

```diff
@@ -1,9 +1,18 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
+  has_one :surveyor
+
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable,
          :jwt_authenticatable, jwt_revocation_strategy: JwtDenylist
+
+  def jwt_payload
```

Comment:
We need to store the role + email in the JWT so that the user can be automatically logged back in on refresh, the ID of the user alone (from `sub`) wasn't cutting it. This more or less resolves #324 since it serves the same purpose.

Reply:
TODO: add `surveyor_id` to this